docs/ruleset_functions/compare_schedules.md

```diff
@@ -0,0 +1,46 @@
+
+## compare_schedules
+
+Description: This function would compare two schedules and determine if they match with or without a comparison factor when applicable.  
+
+Inputs:
+- **schedule_1**: First schedule.
+- **schedule_2**: Second schedule.
+- **mask_schedule**: The schedule that defines comparison mode for all 8760 hours in a year, i.e. if hourly value is 0, schedule_1 shall be equal to schedule_2; if hourly value is 1, schedule_1 shall be equal to schedule_2 times the comparison factor; if hourly value is 2, skip comparison for that hour;  
+- **comparison_factor**: The target multiplier number for schedule_1 compared to schedule_2, i.e. when applicable, the hourly value in schedule_1 shall be equal to that in schedule_2 times the comparison_factor.
+
+Returns:
+- **compare_schedules_result_array**: An array containing the total number of hours that the function compares and the number of hours schedule_1 matches schedule_2 with the comparison_factor, i.e. [total_hours_compared, total_hours_match]
+
```

Comment:
@xingcx i think it would be helpful to add a more detailed description of each of the return values:
1. TOTAL_HOURS_COMPARED: Number of hours that the function compared: the number of hours for which the mask_schedule !=2.
2. TOTAL_HOURS_MATCH: Number of hours for which hourly values in schedule_1 and schedule_2 are in correct relationship
3. EFLH_DIFFERENCE: The ratio of Schedule 1 EFLH to Schedule 2 EFLH for all hours that were compared (hours for which the mask_schedule !=2

Reply:
@mkarpman The latest RDS returns all 3 values you mentioned above. https://github.com/pnnl/ruleset-checking-tool/blob/RDS/XC/function_compare_schedules/docs/ruleset_functions/compare_schedules.md

> compare_schedules_result_dictionary: A dictionary containing the total number of hours that the function compares, the number of hours schedule_1 matches schedule_2 with the comparison_factor, EFLH difference for schedule_1 and schedule_2, i.e. {"TOTAL_HOURS_COMPARED": total_hours_compared, "TOTAL_HOURS_MATCH": total_hours_match, "EFLH_DIFFERENCE: EFLH_difference}